wallet/wallet.c

```diff
@@ -15,6 +15,35 @@
 #define SQLITE_MAX_UINT 0x7FFFFFFFFFFFFFFF
 #define DIRECTION_INCOMING 0
 #define DIRECTION_OUTGOING 1
+/* How many blocks must a UTXO entry be buried under to be considered old enough
+ * to prune? */
+#define UTXO_PRUNE_DEPTH 144
+
+static void outpointfilters_init(struct wallet *w)
+{
+	sqlite3_stmt *stmt;
+	struct utxo **utxos = wallet_get_utxos(NULL, w, output_state_any);
+	struct bitcoin_txid txid;
+	u32 outnum;
+
+	w->owned_outpoints = outpointfilter_new(w);
+	for (size_t i = 0; i < tal_count(utxos); i++)
+		outpointfilter_add(w->owned_outpoints, &utxos[i]->txid, utxos[i]->outnum);
+
+	tal_free(utxos);
+
+	w->utxoset_outpoints = outpointfilter_new(w);
+	stmt = db_prepare(w->db, "SELECT txid, outnum FROM utxoset WHERE spendheight = ?");
+	sqlite3_bind_null(stmt, 1);
+
+	for (size_t i=0; sqlite3_step(stmt) == SQLITE_ROW; i++) {
```

Comment:
`i` seems unused? Why not simply `while (sqlite3_step(stmt) == SQLITE_ROW)`?

Reply:
Greta catch, this was a copy-paste leftover